sdk/core/Azure.Core/tests/common/Internal/ModelSerializerHelper.cs

```diff
@@ -11,17 +10,17 @@ namespace Azure.Core.Tests.Common
     internal static class ModelSerializerHelper
     {
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static void ValidateFormat<T>(IModel<T> model, ModelReaderWriterFormat format)
+        public static void ValidateFormat<T>(IModel<T> model, string format)
         {
             bool implementsJson = model is IJsonModel<T>;
-            bool isValid = (format == ModelReaderWriterFormat.Json && implementsJson) || format == "W";
+            bool isValid = (format == "J" && implementsJson) || format == "W";
```

Comment:
Curious: would it be valid to do 
```suggestion
            bool isValid = (format == ModelReaderWriterOptions.Json.Format && implementsJson) || format == ModelReaderWriterOptions.Json.Format;
```

Asking for two reasons:
- To see if I'm understanding how this works correctly
- Because if we can, I think that'd be preferable to avoid "magic numbers" type errors

Reply:
We would need to duplicate the constants in all libraries since all libraries will have models which will need to compare against these values.

This is one of the "downsides" to removing the ModelReaderWriterFormat, but it removes ambiguity around the conceptual formats `wire` and `data` and the concrete data interchange formats `json`, `xml`, etc.